### DISCUSSION TITLE
How to Add Repository from New Computer

### DISCUSSION THREAD

**Discussion by Mashimaro7** [community/community#22763] [New to GitHub]
<p>I know this is probably a super basic thing, but I just can’t figure it out. I got a new computer, so I downloaded my repository from Github, now I want to back it up again, but if I click add repository, it just says,“This does not appear to be a GitHub repository, would you like to create one here instead?”</p>
<p>How do I add this repository? I didn’t delete the Git files or anything.</p>

**Comment by airtower-luna** [comment]
<p>There’s some important information missing, I’m afraid.</p>
<aside class="quote no-group" data-username="Mashimaro7" data-post="1" data-topic="135775">
<div class="title">
<div class="quote-controls"></div>
 Mashimaro7:</div>
<blockquote>
<p>I got a new computer, so I downloaded my repository from Github</p>
</blockquote>
</aside>
<p>What do you mean by “download”? To me that sounds a bit like you might have downloaded a ZIP archive instead of cloning. If that’s the case you didn’t create a local git repository, you just got the files.</p>
<aside class="quote no-group" data-username="Mashimaro7" data-post="1" data-topic="135775">
<div class="title">
<div class="quote-controls"></div>
 Mashimaro7:</div>
<blockquote>
<p>if I click add repository, it just says,“This does not appear to be a GitHub repository, would you like to create one here instead?”</p>
</blockquote>
</aside>
<p>That sounds like you’re using a git GUI. If you add which one maybe someone who’s familiar with it can help (probably not me, I use the command line almost exclusively).</p>

**Comment by Mashimaro7** [comment]
<p>I’m pretty sure I cloned the repository, but I’m not sure.</p>
<p>I use the GitHub Desktop application.</p>

**Comment by airtower-luna** [comment]
<p>Thank you! I’ve moved the topic to the “How to use Git and Github” section, I hope someone who’s familiar with Github Desktop and can help will see it there.</p>

**Comment by Mashimaro7** [comment, marked as answer]
<p>Okay, well I was able to figure it out on my own, I cloned it again, this time from the GitHub program, and  just moved all my project files from the current build into that new folder and it allowed me to commit.</p>
